herbrain/pregnancy/app.py

```diff
@@ -260,7 +271,6 @@ def my_app(cfg, data="hipp"):
             page=FunctionComponent(
                 ai_hormone_prediction,
                 mesh_explorer=mesh_explorer,
-                show_legend=True if data == "hipp" else False,
```

Comment:
this line is kind of important, as it controls hippocampal substructures legend appearance if data is `hipp`. by removing it, it will appear every time, though it does not make sense for other structures.


EDIT: I've seen this part of the code was deleted below. Should it though? I believe it is a nice to have if we look at hipp with substructures.